ansible_collections/arista/avd/tests/unit/filters/test_range_expand.py

```diff
@@ -25,6 +25,22 @@
     "65100.0-4",
     "65100.0-2,65200.1-2",
     "1-2.0-1",
+    "Ethernet(1)",
+    "Ethernet(1,2,3)",
+    "Ethernet(1)/(4,5)",
+    "(1,2)",
+    "1,(2,3)/5",
+    "Eth(10-14)",
+    "Eth(101-103)/(5,7)",
+    "Eth(1,2",
+    "Eth(1,2)/3,4)",
```

Comment:
Same is the case with this one.. 
Since the syntax is invalid it will pass through and return prefix + split(",") for the invalid part and expand for the valid part which is the first half of the input "Eth(1,2)/3"
So, for "Eth(1,2)/3,4)" --> ["Eth1/3", "Eth2/3", "Eth4)"]

Reply:
I don't follow this logic.
The `(1,2)` should be the only expansion here, since it is the only expression matching the syntax.
So I would expect `["Eth1/3,4)", "Eth2/3,4)"]`.
If we wish, we can indeed decide to implement checks for the unmatched parentheses.

Reply:
Any "," outside the parentheses are used for splitting the string.
For Ex. 
If input is "Eth2/3,4" it will return ["Eth2/3", "Eth4"] since we take the prefix of previous port if not given.
in similar fashion "Eth(1,2)/3, 4)" will return ["Eth1/3", "Eth2/3", "Eth4)"]

Reply:
You are correct.